docs/fast_data/concepts/data_loading.mdx

```diff
@@ -0,0 +1,472 @@
+this means that other tables not involved in the operation will not receive updates during the operation.
+:::
+
+### Scenarios
+
+This operation is needed when:
+
+- the Fast Data is initialized, to make it consistent with the source system;
+- there has been some _disalignment_ between the source system and Fast Data, due to either
+  infrastractural problems or misconfigurations
+
+### Best Practices
+
+Since Initial Load may involve tables having huge size, is necessary to prepare your infrastructure accordingly 
+and stop the generation of Single Views.
+
+:::tip
+If you need to Bootstrap either Fast Data services or infrastructural component, you can check out [this section](/fast_data/faq/bootstrapping_fast_data.md#what-should-be-the-required-resources-to-perform-an-initial-load). 
+:::
+
+Before an Initial Load takes place, you have to prepare your [Fast Data architecture](/fast_data/concepts/architecture.md#putting-it-all-together) accordingly.
+
+<Tabs groupId='fast-data-architecture'>
+
+<TabItem value="event-driven" label="Event-Driven Architecture" default>
+
+First, you have to check that ingestion topics have been correctly defined for the Projections that needs to be populated.
+
+Then, if there exists a Single View Trigger Generator that consumes the messages produced into the `pr-update` topics of
+the projections, you have to check that message consuming is not up-and-running (either by using Runtime Management or setting replicas to zero).
+
+In this way, during the ingestion phase, strategies will not be executed. Once the ingestion procedure has been completed, you can restart
+the SVTG, so that strategies will generate the single view identifiers accordingly.
+
+##### Using Runtime Management
+
+With Runtime Management, is possible to manage Initial Load operations without altering your configuration.
+
+From the Control Plane frontend, you have to check first that the Domain Pipeline that will consume messages
+is in a running state. Then, in the Single View pipeline that will receives data streams
+from the projections involved in the Initial Load, both `Collected` and `Curated` stages must be in a __Paused State__.
+
+The state of your pipeline should match the example shown below, where a data stream generating a single view 
+named `orders-history` is completely paused.  
+
+![Initial Load Runtime Management](img/data_loading_initial_load_runtime.png)
+
+Then, once the Initial Load has been completed and messages have been stored correctly on MongoDB, you can put the `Collected`
+stage back to the __Running State__ and let the underlying SVTG consuming the `pr-update` messages to generate the `trigger` messages.
+
+![Initial Load Runtime Management](img/data_loading_initial_load_runtime_restart_collected.png)
+
+:::tip
+If the Initial Load involves projections having large datasets, is suggested to __only__ restart the base projection, i.e. the projection
```

Comment:
may have a link to base projection documentation details?

Reply:
link added [here](https://github.com/mia-platform/documentation/pull/1391/commits/730e00d01bbf09241b3c687c05224456843836a7)